<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProfileDialogClass</class>
 <widget class="QWidget" name="EditProfileDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditProfileDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,0,2">
   <item>
    <layout class="QGridLayout" name="gridLayout_3" columnminimumwidth="200,0">
     <item row="2" column="0">
      <widget class="QLabel" name="captainLabel">
       <property name="text">
        <string>Капитан</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="isLockedCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="isLockedLabel">
       <property name="text">
        <string>Заблокировать</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="fullFlexLabel">
       <property name="text">
        <string>Флекс</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="fullFlexCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="squireLabel">
       <property name="text">
        <string>Сквайр</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="captainCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="squireCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="identityLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Профиль</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,5,5,5">
     <item>
      <widget class="QLabel" name="statsLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Параметры</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="tankLineLayout" stretch="2,2,2,4,4,5">
       <item>
        <widget class="QLabel" name="tankLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tankRankLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="isActiveTankCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="firstPropTankCheckBox">
         <property name="text">
          <string>мейн</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="secondPropTankCheckBox">
         <property name="text">
          <string>офф</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rankTankLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="dpsLayout" stretch="2,2,2,4,4,5">
       <item>
        <widget class="QLabel" name="dpsLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dpsRankLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="isActiveDpsCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="firstPropDpsCheckBox">
         <property name="text">
          <string>хитскан</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="secondPropDpsCheckBox">
         <property name="text">
          <string>проджектайл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rankDpsLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="supLayout" stretch="2,2,2,4,4,5">
       <item>
        <widget class="QLabel" name="supLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="supRankLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="isActiveSupCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="firstPropSupCheckBox">
         <property name="text">
          <string>хил</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="secondPropSupCheckBox">
         <property name="text">
          <string>второй хил</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rankSupLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
